main/tiff/APKBUILD

```diff
@@ -3,20 +3,23 @@
 # Maintainer: Michael Mason <ms13sp@gmail.com>
 pkgname=tiff
 pkgver=4.0.9
-pkgrel=1
+pkgrel=2
 pkgdesc="Provides support for the Tag Image File Format or TIFF"
-url="http://www.libtiff.org/"
+url="http://www.libtiff.org"
 arch="all"
-license="GPL"
+license="libtiff"
```

Comment:
Thought it bsd-like but it has own one
- https://spdx.org/licenses/libtiff.html
- https://fedoraproject.org/wiki/Licensing:Libtiff?rd=Licensing/libtiff

Reply:
Yes, [libpng](https://github.com/alpinelinux/aports/pull/3865) as well.